README.md

```diff
@@ -18,9 +22,12 @@ ggplot(d, aes(x=x, y=y, country=country, size=x)) +
 
 ![](README_files/figure-markdown_github/demo-1.png)
 
```

Comment:
This looks to be like a weird R Markdown rendering glitch, so I might just revert this range if it's all right with you, @RichardMN